sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -135,7 +135,11 @@ def options(self, rqmt):
             pass  # there is no option in SLURM?
 
         if rqmt.get('gpu', 0) > 0:
-            out.append('--gres=gpu:%s' % rqmt.get('gpu', 0))
+            gres = '--gres=gpu:'
+            if rqmt.get('gpu_name', ''):
+                gres += rqmt.get('gpu_name', '') + ':'
```

Comment:
is the syntax `--gres=gpu:{gpu_name}:{num_gpu}` vanilla slurm or our custom implementation?